reana_client/api/client.py

```diff
@@ -896,7 +939,12 @@ def mv_files(source, target, workflow, access_token):
 
 
 def get_workflow_disk_usage(workflow, parameters, access_token):
-    """Display disk usage workflow."""
+    """Display disk usage workflow.
+
+    :param workflow: name or id which identifies the workflow.
```

Comment:
```suggestion
    :param workflow: name or id of the workflow.
```
What do you think about this, to be consistent with the rest?